### ISSUE TITLE
Accessibility issues highlighter

### ISSUE THREAD

**Issue by Remi05** [Remi05/uial#40]
By knowing the interactions available in a context along with the specific UI elements involved, we can identify if some of those UI elements are missing useful information for the UI to be accessible. Pointing out those issues can help developers improve their interface and make it more accessible.

**Comment by Remi05** [comment]
Looks like [Accessibility Insights](https://accessibilityinsights.io/) is an existing tool for this.
